tools/user_migration/migrate_user.py

```diff
@@ -219,7 +226,8 @@ def create_migration_table(database):
             """CREATE TABLE IF NOT EXISTS
                 migration (
                     fxa_uid VARCHAR(255) NOT NULL PRIMARY KEY,
-                    started BIGINT NOT NULL
+                    started BIGINT NOT NULL,
```

Comment:
This was the other change:

```suggestion
                    started_at BIGINT NOT NULL,
```

and elsewhere in the file